tests/unit/common/spatial/test_point.py

```diff
@@ -16,37 +16,35 @@
 # limitations under the License.
 
 
-from unittest import TestCase
+import pytest
 
 from neo4j._spatial import (
     Point,
     point_type,
 )
 
 
-class PointTestCase(TestCase):
+class PointTestCase:
 
-    def test_wrong_type_arguments(self):
-        for argument in (("a", "b"), ({"x": 1.0, "y": 2.0})):
-            with self.subTest():
-                with self.assertRaises(ValueError):
-                    Point(argument)
+    @pytest.mark.parametrize("argument", ("a", "b"), ({"x": 1.0, "y": 2.0}))
+    def test_wrong_type_arguments(self, argument):
+        with pytest.raises(ValueError):
+            Point(argument)
 
-    def test_number_arguments(self):
-        for argument in ((1, 2), (1.2, 2.1)):
-            with self.subTest():
-                p = Point(argument)
-                assert tuple(p) == argument
+    @pytest.mark.parametrize((1, 2), (1.2, 2.1))
```

Comment:
Have another look at the docs of `pytest.mark.parametrize` https://docs.pytest.org/en/latest/how-to/parametrize.html

You're missing the first argument which defines the name of the parameter. Same mistake in other places, too.